templates/overrides/search/form--navigation-right--oe-whitelabel-search-form.html.twig

```diff
@@ -0,0 +1,32 @@
+{#
+/**
+ * @file
+ * Search form template for the navigation right region.
+ *
+ * @see ./core/modules/system/templates/form.html.twig
+ */
+#}
+{# todo button drupal attributes #}
+{% set submit_button = pattern('button', {
+  'variant': 'light',
+  'icon': 'search',
+  'type': 'submit',
+  'attributes': {
+    'id': 'submit',
+    'class': [
+      'border-start-0',
+      'rounded-0',
+      'rounded-end',
+      'px-3',
+    ],
+  },
+})%}
+
+{% set submit = element.submit|merge({
+  '#markup': submit_button|render,
+  })
+%}
+
+<form {{ attributes.addClass('d-flex', 'mt-3', 'mt-lg-0') }}>
```

Comment:
Also, since above is not needed all you do it add these classes, you dont need to redeclare the file for that, just extend
```
{% extends "form.html.twig" %}
{% set attributes = attributes.addClass('d-flex', 'mt-3', 'mt-lg-0') %}
```